plugins/woocommerce-blocks/bin/gen-block-list-doc.js

```diff
@@ -148,14 +149,18 @@ function readBlockJSON( filename ) {
 		if ( supportsAugmented ) {
 			supportsList = processObjWithInnerKeys( supportsAugmented );
 		}
+
+		if ( blockjson.supports.inserter === false ) {
+			suffix = ' _(Unavailable in inserter)_';
```

Comment:
Is this one needed? Because it is already shown in the `Supports` section with `~~inserter~~` crossed out if not available.

Reply:
I think it's important information to highlight and IMO all strikethrough `supports` properties are super unclear and unintuitive. Especially `~~inserter~~` which you need to find among other data, know what that property means, and then understand the "negation".

<img width="732" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/840232c5-a253-4198-819b-a9dde0c08d21">

That's why I thought it may make sense to highlight this information. But I'm ok with removing that as well. Let me know what you think.

Reply:
The only reason I am against this:

- It gives more priority for `inserter` compared to the other supports and one might argue `multiple` is just as important and why it isn't shown in big text next to the title.
- GB also doesn't highlight this https://developer.wordpress.org/block-editor/reference-guides/core-blocks/#pattern
- There would be redundant information.

Perhaps an alternative is to show a `legend` at the top of the document to highlight what crossed out supports mean to be super clear?

Reply:
> It gives more priority for inserter compared to the other supports and one might argue multiple is just as important and why it isn't shown in big text next to the title.

Got you @roykho . I agree and reverted this change.

> Perhaps an alternative is to show a legend at the top of the document to highlight what crossed out supports mean to be super clear?

I skip that. Let's keep it as is, especially since it's aligned with Guteberg. And we can focus on improving the readability and clarity in scope of https://github.com/woocommerce/woocommerce/issues/48178.